CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ Release report: TBD
 - Fix a regex error in the FIM integration tests ([#3061](https://github.com/wazuh/wazuh-qa/issues/3061)) \- (Framework + Tests)
 - Fix an error in the cluster performance tests related to CSV parser ([#2999](https://github.com/wazuh/wazuh-qa/pull/2999)) \- (Framework + Tests)
 - Fix bug in the framework on migration tool ([#4027](https://github.com/wazuh/wazuh-qa/pull/4027)) \- (Framework)
+- Fix an error related to logs format in reliability test ([#4387](https://github.com/wazuh/wazuh-qa/pull/4387)) \- (Tests)
```

Comment:
Must be placed at the top of the list.

Reply:
Done in [29c4c63](https://github.com/wazuh/wazuh-qa/pull/4387/commits/29c4c630aeaf7e92e62c944c5f931f44273d3282)